coding_schemes/Inclusion_Yes_No.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
This file is called yes_no but doesn't contain the codes yes/no?

Reply:
It's what the variable is called in rapidpro, there's a separate file for yes_no

Reply:
Is there a more appropriate term we can use that matches what RDA would call this?
For projects I've worked on, something like 'Inclusion_Reason' would usually be appropriate, but I'm not sure what terminology RDA are using on this one